### PR TITLE
[PowerPC] Update V18.1.0 release notes

### PR DESCRIPTION
**Pull request by maryammo**
Adding PowerPC updates for clang and llvm into the V18.1.0 release notes.

### CODE REVIEW

**Review by amy-kwan — APPROVED** [review]
LGTM.

**Review by lei137 — APPROVED** [review]
Just some nits for spacing.  Otherwise LGTM
Thx

File: llvm/docs/ReleaseNotes.rst [hunk 1]
```diff
@@ -163,6 +163,30 @@ Changes to the MIPS Backend
 
 Changes to the PowerPC Backend
 ------------------------------
+* LLJIT's JIT linker now defaults to JITLink on 64-bit ELFv2 targets.
```
[the remark this answers is not in the document]
Comment (maryammo) [reply llvm/docs/ReleaseNotes.rst:167]: done

File: llvm/docs/ReleaseNotes.rst [hunk 2]
```diff
@@ -327,6 +352,12 @@ Changes to the LLVM tools
 
 * llvm-objcopy now supports ``--gap-fill`` and ``--pad-to`` options, for
   ELF input and binary output files only.
+* Supported parsing XCOFF auxiliary symbols in obj2yaml.
```
[the remark this answers is not in the document]
Comment (maryammo) [reply llvm/docs/ReleaseNotes.rst:356]: done

**Comment by tstellar** [comment]
Is this ready to merge?

**Comment by maryammo** [comment]
> Is this ready to merge?

Yes it is ready, thank you.

**Pull request merged**